### DISCUSSION TITLE
Will using SerialTelnet solve connection problems?

### DISCUSSION THREAD

**Discussion by AshaiRey** [arkypita/LaserGRBL#1478] [General]
Hi,

I have massive connection problems probably due to the CH340 USB chip on the Nano. I am aware of the problems but all boards that I have use this chips ( Nano, Uno, Wemos D1) and normally it isn't an issue.
Now, I was thinking of using the SerialTelnet option so I don't have to use the USB port (and CH340) anymore.
Will this be a correct assumption before I spend much time in it to rebuild the setup?

**Comment by gmmanonymus111** [comment]
It depends. If you have a stable network with stable WiFi connection, it shouldn't be an issue. I've been using telnet for over a year, without any issues so far. My setup, for reference: [arduino mega + wifi](https://robotdyn.com/mega-wifi-r3-atmega2560-esp8266-flash-32mb-usb-ttl-ch340g-micro-usb.html) (the board with both ATmega and ESP chip), with ramps 1.6, flashed with [GRBL-Mega 5x](https://github.com/fra589/grbl-Mega-5X) and [esp-link](https://github.com/jeelabs/esp-link). (Used [arduino uno + WiFi](https://robotdyn.com/uno-wifi-r3-atmega328p-esp8266-32mb-flash-usb-ttl-ch340g-micro-usb.html) with CNCShield the same way before, also without any issues) Both the machine and my router is in the same room.
EDIT: With this exact setup, you'll still need to use the USB to flash firmware, because OTA update won't work (the ESP resets itself while it tries to reset the arduino, on both boards, but I don't think that'll cause any serious problems on the long run.
